### DISCUSSION TITLE
Display the playstyle (mouse, tablet, touch) used for achieving the score

### DISCUSSION THREAD

**Discussion by GDcheerios** [ppy/osu#14159] [Ideas]
for example
I beat this on tablet
![image](https://user-images.githubusercontent.com/54640414/128610113-ba3cb525-bfb6-4a41-beef-a24be1b06faa.png)
and there would be an icon or something that says tablet

**Comment by frenzibyte** [comment]
Might be possible to have support for that, but likely shouldn't be displayed as a mod.

**Reply by bdach** [reply]
I'm not completely sure what plans there are regarding this going forward, but "Touch Device" is currently very much a (system) mod.

**Reply by frenzibyte** [reply]
If I recall correctly, it also had affects on the scoring, although I can't confirm myself so not sure. But still, feels weird to continue supporting playstyles as "mods" from the beginning.

**Reply by Morodar** [reply]
I think the author used the mod-symbol-shape just as an icon rather than a mod which can be selected?  

Assuming that, he just want to see which input device has been used for that particular score.

**Reply by peppy** [reply]
I think just showing the touch mod would be a good implementation of this.
